src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
```

Comment:
You don't have to comment saying this is from the tutorial. I'd maybe add comments explaining what this class is for though.